build.sbt

```diff
@@ -27,8 +27,8 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 ThisBuild / tlBaseVersion       := "0.48"
 ThisBuild / tlCiReleaseBranches := Seq("master", "scala3")
 
-ThisBuild / scalaVersion       := "3.4.3"
-ThisBuild / crossScalaVersions := Seq("3.4.3")
+ThisBuild / scalaVersion       := "3.5.1"
+ThisBuild / crossScalaVersions := Seq("3.5.2")
```

Comment:
@swalker2m Is there a reason these are different?

Reply:
d'oh, I must have mistyped it